### PR TITLE
Change download URL to official site

### PR DESCRIPTION
**Pull request by rxmichael**
Changed URL to official website per Paw creator's request.

### CODE REVIEW

File: Casks/paw.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   version '2.2.2'
   sha256 '80af2f3bfd973c920d636453f9a9fb8b1d7c3fbd461e72132c2bf12373d06254'
 
-  # amazonaws.com is the official download host per the vendor homepage
-  url "https://luckymarmot-distribution.s3.amazonaws.com/paw/Paw-#{version}.zip"
+  # official download homepage
```
Comment (vitorgalvao) [inline Casks/paw.rb]: This comment must be removed, makes no sense in this case.

**Comment by vitorgalvao** [comment]
Please [amend](http://stackoverflow.com/a/179147/1661012) the commit message to include the name of the updated cask.

> Changed URL to official website per Paw creator's request.

The current URL is just a redirect. Where is said request?

**Comment by teressaeid** [comment]
I am a contributor to the paw team, and this is the official download link, I was asked to update the cask by Micha (Paw’s creator).  

Teressa Eid
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by mittsh** [comment]
[Founder of Paw](https://angel.co/paw) here, this is definitively the best URL to use. The S3 URL may change later (we are actually thinking about having CloudFront CDN).

**Comment by vitorgalvao** [comment]
> I am a contributor to the paw team

Ah, thank you, thought that might be the case. Good enough for me.

> The S3 URL may change later

Please keep in mind, though, that it isn’t unusual for urls to change, and for us to update them when that happens; the eventuality of a link changing isn’t really a strong reason, since it isn’t a big deal to have a temporary sporadic break. However, this isn’t a big deal and the result is the same, so lets do it.

@teressaeid Could you please make the requested changes, so I can merge this? Please @mention me after you do, so I get an email to check back.

**Comment by teressaeid** [comment]
Removed the unnecessary comment.   

Please let me know if there is anything else you want me to do. I am really sorry to bother you with this but this is my first fork and pull request =P !

Thank you!,  

Teressa Eid
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by vitorgalvao** [comment]
> I am really sorry to bother you with this but this is my first fork and pull request =P !

No trouble at all, we’re here for that. Something went wrong, though, as the comment remains. In addition, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by teressaeid** [comment]
Followed the procedure to squash, although it didn’t squash the merges. Really sorry again for the trouble =/ !  

Teressa Eid
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by vitorgalvao** [comment]
Don’t worry, squashing is indeed a bit of an obtuse procedure, and most people have difficulties the first times they try it. I can do that part for you. Thank you for the contribution and picking us for your first PR.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d3cc4a850efd6bff2c8e2fe3f73788c890b5bc62. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by vitorgalvao** [comment]
Merged with fixes as d3cc4a850efd6bff2c8e2fe3f73788c890b5bc62.

**Pull request closed without merging**